.github/actions/akv-secret/index.js

```diff
@@ -1,6 +1,8 @@
 const { spawnSync } = require('child_process');
 const fs = require('fs');
 const os = require('os');
+const path = require('path');
+const { isUtf8 } = require("buffer")
```

Comment:
Do you need a semicolon here?

Reply:
In JavaScript, most semicolons are actually not needed. In this instance, while I agree that it would be more consistent with the remainder of the code's style to have a semicolon, I am loathe to deviate from the tagged v2.49.0.vfs.0.1... Maybe as a `fixup!` on top of e730d37255bba38d14fb909a750a37aa4bd30d91?

Reply:
Pushed on top.